examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>not sure I understand the dataset creation here. In other notebooks we do a short explanation of the dataset, please see: https://github.com/microsoft/recommenders/blob/main/examples/00_quick_start/lstur_MIND.ipynb</p><p> </p><p>Also, there is a step to download the dataset. We have some light wrappers on them like https://github.com/microsoft/recommenders/blob/main/recommenders/datasets/amazon_reviews.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I have elaborated on the data format and dataset creation. Let me know if I need to add more. </p>